javascript/stimulus_reflex.js

```diff
@@ -95,6 +92,34 @@ const invokeCallback = (name, controller) => {
   if (controller && typeof controller[name] === 'function') controller[name]();
```

Comment:
This invokeCallback probably should have a different name because it's useful for all sorts of things that aren't callbacks. Can we call it `send` as a homage to Ruby?

Is `send` a valid function name in JS?

Probably not super important, but technically we could change the name of the controller parameter to class (or klass) because really this is a cool pattern even if you're working in React. I always like the idea that people reading the code can learn new tricks, and this isn't specifically a Stimulus thing.

Of course, it might just be confusing, too. I'm not hung up either way.